pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -446,6 +446,264 @@
         "alpine.3.4.3-x64": {
             "#import": [ "alpine.3.4.3", "alpine.3-x64" ]
         },
+
+    //************************************************
```

Comment:
I'm not sure what if anything will break but in general comments aren't support in json files so you should remove this.